staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -376,6 +377,18 @@ var (
 			},
 		},
 	)
+	SuggestedFlowSchemaHealthForStrangers = newFlowSchema(
+		"health-for-strangers", "exempt", 1000,
+		"",
+		flowcontrol.PolicyRulesWithSubjects{
+			Subjects: groups(user.AllUnauthenticated),
+			NonResourceRules: []flowcontrol.NonResourcePolicyRule{
+				nonResourceRule(
+					[]string{flowcontrol.VerbAll},
+					[]string{"/healthz", "/livez", "/readyz"}),
```

Comment:
`/metrics` should be in the list, is it?

Reply:
how about the other health sub-checks? consider `/[healthz|livez|readyz]/*` in the list?

Reply:
This was a draft of a response to #93359 , which specified exactly the paths listed here.

Reply:
We don't want the detailed health check endpoints to be exempt, since they aren't in any of the critical health-checking paths (i.e. control-plane kubelet liveness-readiness probing).